pkgs/applications/networking/p2p/qbittorrent/default.nix

```diff
@@ -40,11 +40,7 @@ mkDerivation rec {
     ++ optional (!webuiSupport) "--disable-webui"
     ++ optional debugSupport "--enable-debug";
 
-  postInstall = "wrapProgram $out/bin/${
-    if guiSupport
-    then "qbittorrent"
-    else "qbittorrent-nox"
-  } --prefix PATH : ${makeBinPath [ python3 ]}";
+  qtWrapperArgs = optional trackerSearch "--prefix PATH : ${makeBinPath [ python3 ]}";
```

Comment:
I think we want to do that as a bash variable and export that in preFixup IIRC.

Reply:
> I think we want to do that as a bash variable and export that in preFixup IIRC.

Why?

I did as `doc/languages-frameworks/qt.section.md` suggests.